test/runtests.jl

```diff
@@ -56,20 +37,4 @@ include("nlp.jl")
 include("generate_and_solve.jl")
 include("print.jl")
 include("operator.jl")
-include("macros.jl")
-
-# Fuzzer of macros to build expressions
-#include("fuzzer.jl")
-
-# Solver-dependent tests
-#include("model.jl");        length(   lp_solvers) == 0 && warn("Model tests not run!")
-#include("probmod.jl");      length(   lp_solvers) == 0 && warn("Prob. mod. tests not run!")
-#include("callback.jl");     length( lazy_solvers) == 0 && warn("Callback tests not run!")
-#include("qcqpmodel.jl");    length( quad_solvers) == 0 && warn("Quadratic tests not run!")
-#include("nonlinear.jl");    length(  nlp_solvers) == 0 && warn("Nonlinear tests not run!")
-#                            length(minlp_solvers) == 0 && warn("Mixed-integer Nonlinear tests not run!")
-#include("sdp.jl");          length(  sdp_solvers) == 0 && warn("Semidefinite tests not run!")
-#include("socduals.jl");     length(conic_solvers_with_duals) == 0 && warn("Conic solvers with duals tests not run!")
-
-# hygiene.jl should be run separately
-# hockschittkowski/runhs.jl has additional nonlinear tests
+include("macros.jl")
```

Comment:
Which text editor still does not end its files by `\n` :-P

Reply:
Hah I'm not sure sure how that happened!